socorro/schemas/socorro-data-1-0-0.schema.yaml

```diff
@@ -10,6 +10,12 @@ description: >
   Schema for defining Socorro-related data schemas.
 
 definitions:
+  simple_permissions:
+    enum:
+      - "public"
+      - "protected"
+    type: string
```

Comment:
This makes it easier to add permission fields in the schema. Now it's defined in one place with all the options.